src/applications/discharge-wizard/components/FormQuestions.jsx

```diff
@@ -70,19 +70,21 @@ const FormQuestions = ({ formValues, updateFormField }) => {
   const handleScrollTo = e => {
     e.preventDefault();
 
+    const name = e.target.getAttribute('name');
```

Comment:
Excellent refactoring, love it when we can use short names instead of chained names over and over.